pages/common/shellcheck.md

```diff
@@ -1,20 +1,33 @@
 # shellcheck
 
 > Shell script static analysis tool.
-> More information: <https://www.shellcheck.net/>.
+> Check shell scripts for errors, usage of deprecated/insecure features, and bad practices.
+> More information: <https://www.shellcheck.net>.
 
 - Check a shell script:
 
-`shellcheck {{file.sh}}`
+`shellcheck {{path/to/script.sh}}`
 
-- Override script's shebang:
+- Check a shell script interpreting it as the specified shell dialect (overrides the shebang at the top of the script):
 
-`shellcheck --shell {{sh|bash|ksh}} {{file.sh}}`
+`shellcheck --shell {{sh|bash|dash|ksh}} {{path/to/file.sh}}`
 
-- Ignore certain errors:
+- Ignore one or more error types:
 
-`shellcheck --exclude {{SC1009}} {{file.sh}}`
+`shellcheck --exclude {{SC1009,SC1073}} {{path/to/file.sh}}`
```

Comment:
Same here.

Reply:
consistency, `path/to/script.sh` or `path/to/file.sh` @einverne , in your other comment you say script :P